docs/docs/config/backlight.md

```diff
@@ -34,7 +34,7 @@ Applies to: [`/chosen` node](https://docs.zephyrproject.org/latest/build/dts/int
 
 See the Zephyr devicetree bindings for LED drivers:
 
-- [gpio-leds](https://docs.zephyrproject.org/latest/build/dts/api/bindings/gpio/gpio-leds.html)
+- [gpio-leds](https://docs.zephyrproject.org/latest/build/dts/api/bindings/led/gpio-leds.html)
```

Comment:
While we are at it, can we instead link to the 3.0 release version docs, so we're not looking to a moving target with the `latest` piece?

Reply:
I wasn't sure if you wanted to use the 3.0.0 or use whatever the latest is for 3.0.x. If this isn't good, I'll have to figure out a more clever way to always be selecting 3.0.x.

Reply:
Fixed at 3.0.0 is fine, it should be API stable for 3.0.x.